test_vm/src/lib.rs

```diff
@@ -455,7 +456,7 @@ impl<'bs> VM<'bs> {
             policy: &Policy::default(),
             subinvocations: RefCell::new(vec![]),
             actor_exit: RefCell::new(None),
-            read_only: false,
+            flags: SendFlags::default(),
```

Comment:
Let's revert this. In the future, this may end up being https://github.com/filecoin-project/ref-fvm/blob/50105fc54107db31d46077c7a6a03c79584c5e72/shared/src/sys/out.rs#L71.

These are _separate_ types because they may end up containing separate bits.